NBA/reformat.py

```diff
@@ -0,0 +1,9 @@
+def reformat(df,id_name):
```

Comment:
Please add comment and rename the function to be more explicit. 

What is it renaming?